### DISCUSSION TITLE
Issue with Negative Faction Power Value Post-Migration from v4 to v5

### DISCUSSION THREAD

**Discussion by tekikaito** [Dans-Plugins/Medieval-Factions#1753] [Q&A] [answered]
Hello Everyone,

Firstly, I'd like to express my gratitude for this exceptional plugin; it's been a great asset.

Recently, I encountered an issue following a migration from v4 to v5. For some reason, one faction on my server is now displaying a negative power value. I suspect this may have something to do with the transition from JSON files to a database (DB) system.

I've tried to diagnose the issue using DBeaver to connect to the DB and inspect the values, but I was unsuccessful. The following error was returned:

com.dansplugins.factionsystem.shadow.org.h2.mvstore.tx.VersionedValueType$Factory

The DB settings remain at their default values, and this power level inconsistency is unique to one specific faction.

Does anyone have any insight or advice on how to resolve this issue? Any guidance would be much appreciated.

Best Regards,
Squawnchy

**Comment by dmccoystephenson** [comment, marked as answer]
Hey @tekikaito,

Was this ever resolved?

Signed,
Daniel

**Reply by tekikaito** [reply]
Hey @dmccoystephenson 

I resolved the issue by completely reinstalling the plugin and using a mysql db instead. There seems to be a mechanism that is decreasing player power while they are offline, am I right? I think that is what was causing the negative values. I now have set minPower to 250 so the players stop losing their power, but it would be nicer if I could just disable or slow down the power decrease or maybe I just misconfigured the plugin. I dont know.

Anyway - I think you can close this.
